src/stores/views.ts

```diff
@@ -680,7 +691,14 @@ export function viewAddFilterItemsToRow(
   let newRows: Row[] = timelines[0].rows;
   let returnRow: Row | undefined = undefined;
   const defaultRowName = `${capitalize(typeName)} Row`;
-  const row = typeof rowId === 'number' ? newRows.find(r => r.id === rowId) : undefined;
+  // If no row was given, but one matches the default name, attempt to use it
+  const row =
+    typeof rowId === 'number'
+      ? newRows.find(r => r.id === rowId)
+      : newRows.find(iterRow => {
+          return iterRow.name === defaultRowName;
+        }) || undefined;
+
```

Comment:
Is this default name logic needed? Seems a little brittle

Reply:
Agreed - this is a remnant of the code to add external event types to a layer's filter when added which is now re-worked & doesn't exist. Reverting this line back to it's `develop` version